builders/build/substrate/adding-built-in-pallet.md

```diff
@@ -0,0 +1,192 @@
+
+In this example, some parameters have a description attached. For a detailed description of each parameter, refer to the [official config trait for the Assets module documentation](https://paritytech.github.io/substrate/master/pallet_assets/pallet/trait.Config.html){target=_blank}.
+
+```rust
+...
+parameter_types! {
+   // These constants will be converted to structs that allows the runtime to read their type and value in a standardized way
+
+   // The basic amount of funds that must be reserved for an asset
+	pub const AssetDeposit: Balance = 100;
+   // The amount of funds that must be reserved when creating a new approval
+	pub const ApprovalDeposit: Balance = 1;
+	pub const MetadataDepositBase: Balance = 10;
+	pub const MetadataDepositPerByte: Balance = 1;
+
+   // Maximum lenght for a symbol name
+   pub const StringLimit: u32 = 50;
+}
+
+// Implementing the Assets confif trait for the runtime
+impl pallet_assets::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+   // Stores the balances in an unsigned integer of 128bits
+	type Balance = u128;
+   // The id of an asset can be defined as an unsigned integer of 64 bits
+	type AssetId = u64;
+	type AssetIdParameter = u64;
+   // Uses module Balances as mechanism for currency operations
+	type Currency = Balances;
+   // Defines the allowed origins to create assets
+	type CreateOrigin = frame_support::traits::AsEnsureOriginWithArg<frame_system::EnsureSigned<AccountId>>;
+   // Root can create assets
+	type ForceOrigin = EnsureRoot<AccountId>;
+	type AssetDeposit = AssetDeposit;
+	type AssetAccountDeposit = frame_support::traits::ConstU128<1>;
+	type MetadataDepositBase = MetadataDepositBase;
+	type MetadataDepositPerByte = MetadataDepositPerByte;
+	type ApprovalDeposit = ApprovalDeposit;
+	type StringLimit = StringLimit;
+	type Freezer = ();
+	type Extra = ();
+	type WeightInfo = pallet_assets::weights::SubstrateWeight<Runtime>;
+	type RemoveItemsLimit = frame_support::traits::ConstU32<1000>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type BenchmarkHelper = ();
+   type CallbackHandle = ();
+}
+...
+```
+
+### Add the module to the runtime {: #add-module-to-runtime }
+
+In the same `lib.rs` file referenced in the previous section, there is a segment enclosed in the macro 'construct_runtime!()', this is where the pallet must be added to be included within the runtime. Since the example is based on the EVM template, the runtime is already configured to include many modules, including the modules for system support, the modules to add the Ethereum compatibility layer, the modules to support the Tanssi protocol, balances, and now, also Assets:
```

Comment:
```suggestion
In the same `lib.rs` file referenced in the previous section, there is a segment enclosed in the macro 'construct_runtime!()'. This is where the pallet must be added to be included in the runtime. Since the example is based on the EVM template, the runtime is already configured to include many modules, including the modules for system support, the modules to add the Ethereum compatibility layer, the modules to support the Tanssi protocol, balances, and now also Assets:
```